.github/workflows/remote-build-android-hybrid.yml

```
@@ -0,0 +1,30 @@
+name: Remote Build Android Hybrid
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest-xl
+    steps:
+      - name: Checkout
+        # v4
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
+        with:
+          submodules: true
+          ref: ${{ github.event.pull_request.head.sha }}
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/composite/setupNode
+
+      - name: RNEF Remote Build - Android
+        uses: callstackincubator/android@a49920aadab9f41951944ea52bb7983fa6b20b03
+        env:
+          PROJECT_ROOT_PATH: "Mobile-Expensify/"
+        with:
+          variant: Debug
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

NAB, these are the same thing but I prefer this syntax.

```suggestion
          github-token: ${{ github.token }}
```